src/rebar_http_adapter.erl

```diff
@@ -0,0 +1,41 @@
+%% Derived from hex_core v0.7.1 for extra flexibility.
+
+-module(rebar_http_adapter).
```

Comment:
```suggestion
-module(rebar_httpc_adapter).
```

Better name since it is an adapter to use httpc?

Reply:
Possibly, but I named it after the `http_adapter` hex_core option pattern rather than what it does. I'll rename it.